manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -533,12 +533,13 @@ properties:
         redirect-uri: (( concat "https://login." properties.system_domain ))
       paas-usage-events-collector:
         access-token-validity: 1209600
-        authorized-grant-types: client_credentials,refresh_token
+        authorized-grant-types: client_credentials,refresh_token,authorization_code
+        autoapprove: true
         override: true
         secret: (( grab secrets.uaa_clients_paas_usage_events_collector_secret ))
-        scope: openid,oauth.approvals,cloud_controller.admin_read_only
-        authorities: oauth.login,cloud_controller.admin_read_only
-        redirect-uri: (( concat "https://login." properties.system_domain ))
+        scope: openid,oauth.approvals,cloud_controller.admin_read_only,cloud_controller.read,cloud_controller.global_auditor,cloud_controller.admin
```

Comment:
Why do we need the cloud_controller.admin scope here?

Reply:
We need to determin if someone has access to edit the `pricing_plans`.

Currently anyone with `admin` `admin_readonly` or `global_auditor` roles can edit pricing.

In the follow up story I have made a suggestion that we could create a new scope in UAA called `billing.admin` for this purpose and assign it to only those who should have access (ie Tom).

Reply:
Ok, because the admin access feels here too much.

Reply:
Agreed... In fact I'll make the "invent a billiing.admin role" into it's own story so it doesn't get lost in the list of "nice to haves"